keys/christian.asc

```diff
@@ -0,0 +1,14 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+
+mDMEYvZ90RYJKwYBBAHaRw8BAQdAkxsaalUc8UsOZ23LsziqSbJS8ILYkklzVYzS
```

Comment:
This key is surprisingly short compared to the others 🤔 did you create it following these [instructions](https://docs.github.com/en/authentication/managing-commit-signature-verification/generating-a-new-gpg-key)?

Reply:
hey @otoumas it was a few months back when i started, but yes I do recall going through those instructions. i noticed the algorithm used to generate this key was `ed25519`, which from a quick google search apparently is a smaller key? i haven't had any issues with signing commits with this key, but let me know if it poses any potential issues and i can generate another one.